src/utils/client.js

```diff
@@ -1,20 +1,47 @@
 /* global __TARGET__ */
 
 import { initClient } from 'ducks/authentication/lib/client'
-import CozyClient from 'cozy-client'
+
+import CozyStackClient from 'cozy-stack-client'
+import CozyClient, { StackLink } from 'cozy-client'
+import PouchLink from 'cozy-pouch-link'
+import { schema } from 'doctypes'
 
 export const getClientMobile = persistedState => {
   const hasPersistedMobileStore = persistedState && persistedState.mobile
   return initClient(hasPersistedMobileStore ? persistedState.mobile.url : '')
 }
 
+export const links = {}
+
 export const getClientBrowser = () => {
   const root = document.querySelector('[role=application]')
   const data = root.dataset
-  return new CozyClient({
+
+  const stackClient = new CozyStackClient({
     uri: `${window.location.protocol}//${data.cozyDomain}`,
     token: data.cozyToken
   })
+
+  if (__TARGET__ === 'mobile') {
+    links.pouch = new PouchLink({
+      doctypes: ['io.cozy.bank.operations'],
+      initialSync: true,
+      client: stackClient
+    })
+  }
+
+  links.stack = new StackLink({ client: stackClient })
+
+  const client = new CozyClient({
+    link: [
+      __TARGET__ === 'mobile' && links.pouch,
+      links.stack
+    ].filter(Boolean),
```

Comment:
Is the order in the array important ? i.e. here pouch link will be used first ?

Reply:
yes